src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
It's trying to add the case to allow an empty string, which your example is missing:

```
node -e "let types = [undefined, null, '', 'string']; types.forEach(t => console.dir({[t]: t == null ? true : false}))"
{ undefined: true }
{ null: true }
{ '': false }
{ string: false }
```